cves/CVE-2009-3555.yml

```diff
@@ -4,27 +4,27 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Man in the Middle
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-300 Channel Accessible by Non-Endpoint ('Man-in-the-Middle')
```

Comment:
```suggestion
CWE: 300
```

Reply:
From comments on my own PR, this CWE value should just be the number value as it will be parsed out.